### PR TITLE
:sparkles: typescript support

### PR DESCRIPTION
**Pull request by THMKAE**
ℹ️ Contains Typescript support and small config updates (less strict)

### CODE REVIEW

**Review by RobbeB — COMMENTED** [review]
Isn't this a bit too much blackbox?
Maybe it is better to just provide our own rules, which the user can extend from, instead of hijacking everything from the eslintrc config

File: flowtype.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = require('./packages/flowtype.json');
\ No newline at end of file
+module.exports = require('./packages/flowtype.json');
```
Comment (laurensvc) [inline flowtype.js:1]: mag die file niet weg dan ?

File: packages/core.json [hunk 2]
```diff
@@ -240,10 +188,7 @@
         "position": "above"
       }
     ],
-    "linebreak-style": [
-      2,
-      "unix"
-    ],
+    "linebreak-style": [2, "unix"],
```
Comment (laurensvc) [inline packages/core.json:191]: muh windows

File: packages/core.json [hunk 3]
```diff
@@ -282,10 +224,7 @@
       }
     ],
     "new-parens": 2,
-    "newline-after-var": [
-      0,
-      "always"
-    ],
+    "newline-after-var": [0, "always"],
```
Comment (laurensvc) [inline packages/core.json:227]: not needed imo

File: flowtype.js (hunk 1 above)
Comment (THMKAE) [reply flowtype.js:1]: I prefer not to remove this one, so users can still choose flowtype with a more custom approach.

File: packages/core.json (hunk 3 above)
Comment (THMKAE) [reply packages/core.json:227]: 0 is off (see https://eslint.org/docs/user-guide/configuring#configuring-rules) so it will not throw an error or warning.

**Review by Mcalewaert — APPROVED** [review]

**Pull request merged**